automated-comments/elixir/solution/use_module_doc.md

```diff
@@ -0,0 +1,29 @@
+<!--
+This error is raised when the submission lacks @moduledoc for the Module
+-->
+
+It is recommended to have a [`@moduledoc`][1] attribute to describe the intent
```

Comment:
I'd prefer to not see numerical links, but `[…](…)`-style ones instead. When there are many comments combined into a single one, the footnote-style used here might cause issues.

Reply:
For sure, can change it, I don't know how it all gets compiled by the website and that seems like a good defensive measure.

Reply:
done